doc/tutorials/running_deb_as_serv_vm.rst

```diff
@@ -43,89 +43,93 @@ Use the following instructions to install Debian.
 Validated Versions
 ******************
 
-- **Debian version:** 10.0 (buster)
-- **ACRN hypervisor tag:** acrn-2019w35.1-140000p
-- **Debian Service VM Kernel version:** 4.19.68-84.iot-lts2018-sos
+- **Debian version:** 10.1 (buster)
+- **ACRN hypervisor tag:** acrn-2020w40.1-180000p
+- **Debian Service VM Kernel version: tag:** acrn-2020w40.1-180000p
```

Comment:
I would prefer that we stay consistent with the instructions below, so either we mention the branch (`release_2.2`) here, or we change the instructions below to check out this tag (and not the branch).

Reply:
Done!